### DISCUSSION TITLE
Some old documents (cdms and friends) for newcomers :)

### DISCUSSION THREAD

**Discussion by jypeter** [xCDAT/xcdat#170] [3. Ideas]
This is kind of between a feature and a documentation request

You can visit my [old documentation](https://sharebox.lsce.ipsl.fr/index.php/s/EvtQJZuWR4UBo0X) link if you want to download some of the old CDAT documents. I have lots of other old CDAT documents and tutorials, but I have only uploaded the most important ones

This will give you an idea of the features we use and like. I understand that the xarray based API will make backward compatibility impossible, but the documentation shows the useful (for a climate scientist) features that were available and that will hopefully be replicated in some way

- `cdms_quick_start.pdf`: yes, that file is **20 years old**, that's how **visionary** Bob's work was. Too bad it was never more widely adopted
- `cdms5.pdf`: the last pdf version of the `cdms` (and `cdtime`) documentation (I have not uploaded earlier versions). It's long but very good. I think that's probably what Denis used for [readthedoc CDMS documentation](https://cdms.readthedocs.io/). But there were several conversion errors and it may still be better to use the original pdf
- `cdat_utilities.pdf`: the original documentation of the spatial and time averaging functions we still use regularly (i.e. the original content of `cdutil` and `genutil`)
- `Unlocking_CDATs_Best_Kept_Secrets.pdf`: one of Charles' tutorials. I'd say read pages: 6-10, 13-14, 17 (filters)
- `05_quick_guide_to_read_manip_plot_write.pdf`, `08_averaging_grids_and_interpolation.pdf`, `09_statistical_tools_genutil.pdf`, `10_climate_specific_tools.pdf`, `13_cdtime_time_manipulation.pdf`: more on cdms, cdtime, cdutil, genutil

**Comment by chengzhuzhang** [comment]
Hey @jypeter Thank you for providing these very useful documentation! This issue, as well as the #159 Moving from CDAT to xcdat, will perhaps suit better in `discussion` rather than an `issue` that we can resolve. Do you mind if I convert both to discussion? I think they will be very useful for entraining discussions from newcomers as well.

**Comment by tomvothecoder** [comment]
@jypeter Thanks for this info, we appreciate your input!

Also good idea about using the discussion forum @chengzhuzhang. Just like #159, I will be moving this discussion over to #168.

**Reply by jypeter** [reply]
Thanks @tomvothecoder and @chengzhuzhang for moving this and #169 to the most appropriate place! I confess I did not know about the _discussion_ feature of github (and have never used it before)
